### PR TITLE
Fixture

### PR DESCRIPTION
**Pull request by PalumboN**
Adding Fixture node and fields, fixture and methods to Describe.

Actually only affects to parser stage, I wanted to touch the interprete but I need help to understand :P

### CODE REVIEW

File: src/model.ts [hunk 1]
```diff
@@ -114,6 +115,7 @@ export type Mixin<S extends Stage> = BaseNode<'Mixin', S> & {
 
 export type ObjectMember<S extends Stage> = Field<S> | Method<S>
 export type ClassMember<S extends Stage> = Constructor<S> | ObjectMember<S>
+export type DescribeMember<S extends Stage> = Field<S> | Fixture<S> | Test<S> | Method<S>
```
Comment (nscarcella) [inline src/model.ts]: If we are not using this anywhere else, we might as well just use the type union on the Describe.members field

File: test/parser.test.ts [hunk 2]
```diff
@@ -1059,6 +1079,28 @@ describe('Wollok parser', () => {
 
   })
 
+  describe('Fixture', () => {
```
Comment (nscarcella) [inline test/parser.test.ts]: You totally got the idea of which scenarios we want to cover. This are the perfect test cases for this feature. Good Job.

File: src/model.ts (hunk 1 above)
Comment (PalumboN) [reply src/model.ts]: It appears here too:
https://github.com/uqbar-project/wollok-ts/pull/11/files#diff-a6322b30cb17d35f81779ff7980ea4c3R32

**Comment by PalumboN** [comment]
@nscarcella check it now!

**Pull request merged**